modules/varnish/templates/mediawiki.conf.erb

```diff
@@ -98,8 +98,8 @@ server {
 	server_name miraheze.org *.miraheze.org;
 	root /var/www/html;
 
-	ssl_certificate /etc/ssl/localcerts/wildcard.miraheze.org-2020-2.crt;
```

Comment:
This will break custom domains using varnish. Cloudflare cert can only be used with Cloudflare.

Reply:
No it won't - they'll use their block - it's an expired cert anyway, we don't need a default block at all tbh though. We should only be serving domains which have explicit configuration. We could remove the miraheze.org & default blocks altogether.